x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/description_step/filter_label.tsx

```diff
@@ -1,95 +0,0 @@
-/*
```

Comment:
I'm not sure I understand why these SIEM files were deleted... could you explain?

Reply:
They duplicated files from our repo, requiring some really internal types that shouldn't be exported.
I instead exported `FilterLabel` and used that.